Gruntfile.js

```diff
@@ -50,6 +50,24 @@ const applyPortShift = (env) => {
   return { ...env, API_PORT, API_URL }
 }
 
+const untilServerIsReady = () => {
```

Comment:
Maybe this should be moved below under the comment that says `// Task functions`?

Reply:
It's true that it's used in some task e.g. `backend:relaunch`, but it's not meant to be directly wrapped into a grunt task (there is no `grunt untilServerIsReady` task).

Reply:
But there is no `grunt copyTo` and `grunt fork2` tasks either, so I'm not sure I understand your reply...

Reply:
Hmm let's move it then. You're right about `copyTo` and `fork2`. I thought they belonged here, but in fact that's not so much consistent.